projects/hslayers/src/components/query/partials/default-info-panel-body.html

```diff
@@ -1,4 +1,4 @@
-<div [hidden]="!(HsQueryBaseService.data.coordinates.length > 0)" class="card hs-main-panel">
+<div [hidden]="!(HsQueryBaseService.data.features.length > 0)" class="card hs-main-panel">
```

Comment:
This might break getfeatureinfo (not display anything) if we dont have any vector features but only WMS layers on map